storage/src/backend/connection.rs

```diff
@@ -165,11 +165,39 @@ impl ProxyHealth {
     }
 }
 
+const UNSET: i16 = 0;
```

Comment:
Can we clarify these constants? for example `SCHEME_CACHE_UNSET`.

Reply:
fixed